<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>status</class>
 <widget class="QWidget" name="status">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zarządzanie statusami</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white;
    color: black;
}
QPushButton QAbstractItemView {
    background-color: white;
    color: black;
}
</string>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white;
    color: black;
}
QPushButton QAbstractItemView {
    background-color: white;
    color: black;
}
</string>
   </property>
   <property name="text">
    <string>Edytuj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white;
    color: black;
}
QPushButton QAbstractItemView {
    background-color: white;
    color: black;
}
</string>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListView {
    background-color: white;
    color: black;
}
QListView QAbstractItemView {
    background-color: white;
    color: black;
}
</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: white;
    color: black;
}
QPushButton QAbstractItemView {
    background-color: white;
    color: black;
}
</string>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
